src/main/java/randoop/util/EquivalenceChecker.java

```diff
@@ -0,0 +1,51 @@
+package randoop.util;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility class for checking equivalence between objects in randoop.generation.Detective. This
```

Comment:
This seems to work only on classes, not arbitrary objects.

Reply:
Should other parts of Randoop also permit wrapping and unwrapping, as you have made the demand-driven approach do?